compiler/src/iree/compiler/Dialect/HAL/IR/HALAttrs.td

```diff
@@ -186,6 +186,14 @@ def HAL_DescriptorSetLayoutFlagsAttr :
   let cppNamespace = "::mlir::iree_compiler::IREE::HAL";
 }
 
+def HAL_DispatchFlags_None : I64BitEnumAttrCase<"None", 0x0000>;
+def HAL_DispatchFlagsAttr :
+    I64BitEnumAttr<"DispatchFlags", "valid Dispatch flags", [
+      HAL_DispatchFlags_None,
+    ]> {
+  let cppNamespace = "::mlir::iree_compiler::IREE::HAL";
```

Comment:
nit: why the capitalization in `valid Dispatch flags`?

Reply:
copy/paste :P fixed